src/app/loginComponents/requests/requests.component.html

```diff
@@ -93,6 +110,15 @@ <h3>My Requests</h3>
             }}</a>
           </mat-card-title>
           <mat-card-subtitle> Requested on {{ membership.dbCreateDate | date }} </mat-card-subtitle>
+          <button
+            mat-flat-button
+            [id]="'delete-my-rejected-org-' + i"
+            class="ml-auto h-50"
+            (click)="openDeleteDialog(membership.organization.displayName, membership.organization.id)"
+            matTooltip="Delete this organization request"
```

Comment:
For here and the other buttons above, this is minor but there's no reason to withhold information:

1. I would mention the status, i.e., Delete this rejected organization
2. Or I would mention the org, e.g., Delete "Acme Explosives" organization

Combining 1 and 2 seems tricky for some reason.

Reply:
Combined 1 and 2.